### DISCUSSION TITLE
Cookie count are not being constant for same website at same time

### DISCUSSION THREAD

**Discussion by prakashkatariyatatvic** [GoogleChromeLabs/ps-analysis-tool#614] [Support Forum] [answered]
While checking the website cookies in privacy sandbox extension, it's showing different count for every test that we perform. For example, when we analyze the example.com first time at 22:10, the total cookie shown by tool is 12 and 2nd time if we clear all cookie and re-analyze the same website at 22:12, the total cookie shown by tool is 17.  Please find the screenshot attached and let me know in case you have better answer to work with this extension which help me take necessary business decision.

**Note:** We are analyzing all the scenarios in new chrome profile to avoid any problems.

**1st time result:**
![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/157360089/7f853205-87d0-4b34-bf6d-eaf020272db6)

**2nd time result**
![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/157360089/7266abba-8a62-49e8-a4d1-3e3149073f1c)

**Comment by milindmore22** [comment, marked as answer]
Hi @prakashkatariyatatvic 

Thanks for reporting the inconsistencies with the PSAT cookie count. You're absolutely right – several factors can affect the reported number of cookies, and it's important to be aware of this. Here's a breakdown of potential reasons for the discrepancies:

**User Interaction**: Certain cookies depend on user interaction before they get set. For example, cookies associated with chat widgets might not appear unless you open the chat box. Similarly, cookies triggered by specific user actions like scrolling or clicking might not show up immediately.

**Location-Based Loading**: Some cookies are dynamically loaded based on your location or region. This means the reported cookie count might differ depending on where you access the website.

**Delayed Loading**: We're aware of a known issue with delayed cookie loading, which can lead to an inaccurate initial count. This issue is being actively addressed in GitHub issue #380, and we'll keep you updated on the progress.

We're constantly working to improve the accuracy and consistency of the PSAT extension. Your feedback is valuable in helping us identify and address potential issues.

Here are some additional things you can do:

**Refresh the page:** Sometimes, refreshing the page can trigger the loading of delayed cookies.

**Try visiting the website from different locations:** If you suspect location-based loading might be playing a role, try accessing the website from another region and see if the cookie count changes.

**Keep an eye on GitHub issue #380:** We'll post updates on the progress of fixing the delayed loading issue there.

If you have any further questions or encounter any specific inconsistencies, please don't hesitate to contact us on PSAT [support forums](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/categories/support-forum). We appreciate your understanding and patience as we work to improve the PSAT extension.

**Comment by milindmore22** [comment]
@prakashkatariyatatvic I will be converting this issue to support forum discussion, For future reference, please [create support discussion](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/new?category=support-forum) if you have any question about the PSAT.
